modules/appeals_api/app/swagger/appeals_api/v2/api_description.md

```diff
@@ -25,7 +25,7 @@ The submission statuses begin with pending and end with caseflow.
 | submitted   | A submitted status means the data was successfully transferred to the central mail portal.<br /><br />A submitted status is confirmation from the central mail portal that they have received the PDF, but the data is not yet being processed. The Date of Receipt is set when this status is achieved.<br /><br />Submitted is the final status in the sandbox environment.<p> |
 | processing   | Indicates intake has begun, the Intake, Conversion and Mail Handling Services (ICMHS) group is processing the appeal data. |
 | success   | The centralized mail portal, Digital Mail Handling System (DHMS), has received the data. |
-| caseflow   | Final status. The data is in the caseflow system and the Appeals Status API can be used to check the status of the appeal in the AMA process. |
+| complete   | Final status. The data is in the caseflow system and the Appeals Status API can be used to check the status of the appeal in the AMA process. |
```

Comment:
I don't know that this means it's in Caseflow. I think the farthest it gets is VBMS. Maybe it's still OK to say this though? Thoughts?

Reply:
Yeah I planned on having us go back through this wording in another ticket.